envpool/sokoban/level_loader.cc

```diff
@@ -206,7 +210,9 @@ std::vector<SokobanLevel>::iterator LevelLoader::GetLevel(std::mt19937& gen) {
   auto out = levels_.begin() + cur_level_;
   cur_level_ += num_envs_;
   levels_loaded_++;
-  return out;
+
+  TaggedSokobanLevel tagged_level = {cur_level_file_, out->first, out->second};
```

Comment:
```suggestion
  TaggedSokobanLevel tagged_level{cur_level_file_, out->first, out->second};
```

will invoke direct-initialization instead of copy-initialization https://en.cppreference.com/w/cpp/language/copy_initialization

I hate C++

Reply:
In this case likely no actual copy would be performed and it's just semantics